docs/organizer-guide/meetups/livestreaming-meetups.rst

```diff
@@ -0,0 +1,40 @@
+Livestreaming or Recording a Meetup
+===================================
+
+Whether due to health or environmental reasons or because you live somewhere where people are spread out over a large area, maybe you want to try live streaming a meetup.
```

Comment:
```suggestion
Whether due to health or environmental reasons, or because you live somewhere where people are distributed across a large area, maybe you want to try live-streaming a meetup.
```